src/test/java/org/prebid/cache/handlers/PostCacheHandlerTests.java

```diff
@@ -161,4 +164,41 @@ void testExternalUUIDInvalid() {
                 .expectComplete()
                 .verify();
     }
+
+    @Test
+    void testUUIDDuplication() {
+        CacheConfig cacheConfigLocal = new CacheConfig(cacheConfig.getPrefix(), cacheConfig.getExpirySec(), cacheConfig.getTimeoutMs(),
```

Comment:
You are using mixed approach in variables naming, below in this test val is used, so replace types with val or val with types.